locales/in.csv

```diff
@@ -0,0 +1,106 @@
+amount: राशि
+ask_world_delete_1: वास्तव में दुनिया हटाना चाहते हैं "
+ask_world_delete_2: "?
+back: वापस
+back_to_game: खेल पर वापस जाएं
+change_world_name: दुनिया का नाम बदलें
+chunk_regenerated: चंक पुनः उत्पन्न हुआ।
+chunks_loaded: चंक्स लोड हो गए।
+commands: कमांड्स
+delete_local: (स्थानीय) हटाएं
+delete_world: दुनिया हटाएं
+delete: हटाएं
+download_world_from_server: सर्वर से दुनिया डाउनलोड करें
+download: डाउनलोड
+error_amount_minimum_1: राशि कम से कम 1 होनी चाहिए!
+error_conflict_1: संघर्ष! दुनिया "
+error_conflict_2: " को यहाँ और कहीं और संशोधित किया गया था।\nठीक: सर्वर से वेरिएंट लें (
+error_conflict_3: ) | रद्द करें: स्थानीय वेरिएंट रखें (
+error_conflict_4: )
+error_connection: कनेक्शन त्रुटि।
+error_delete_world: दुनिया हटाते समय त्रुटि:
+error_download_world: दुनिया डाउनलोड करते समय त्रुटि:
+error_edit_world: दुनिया संपादित करते समय त्रुटि:
+error_gamemode_range: गेम मोड 0..2 के बीच होना चाहिए!
+error_invalid_block_type: अमान्य ब्लॉक प्रकार!
+error_inventory_full: इन्वेंटरी भरी हुई है!
+error_list_is_loading: सूची लोड हो रही है!
+error_loading_worldlist: दुनिया सूची लोड करते समय त्रुटि:
+error_name_too_long: नाम बहुत लंबा है!
+error_no_permission_logged_in: अनुमति नहीं है। लॉग इन किया गया है?
+error_no_permission: कोई अनुमति नहीं!
+error_no_world_selected: कोई दुनिया चयनित नहीं!
+error_not_logged_in: लॉग इन नहीं किया गया!
+error_only_vert_supported: केवल "vert" समर्थित है!
+error_opened: minicraft पहले से चल रहा है!
+error_parameter_missing: पैरामीटर गुम है!
+error_pitch: पिच!
+error_selection_required: चयन आवश्यक है!
+error_storage: अपर्याप्त संग्रहण!
+error_upload_world: दुनिया अपलोड करते समय त्रुटि:
+error_world_is_loading: दुनिया अभी भी लोड हो रही है!
+error_world_is_present_both_sides: दुनिया दोनों तरफ मौजूद है!
+error_world_not_downloaded: दुनिया अभी तक डाउनलोड नहीं हुई है!
+error_world_not_uploaded: दुनिया अपलोड नहीं हुई है!
+game_saved: खेल सहेजा गया।
+gamemode_set_to: गेम मोड सेट किया गया
+inventory_cleared: इन्वेंटरी साफ़ हो गई।
+inventory: इन्वेंटरी
+item_labels: !!!अंग्रेजी में उपयोग नहीं होता!!!
```

Comment:
only in english its not used, other languages need it. look into other language files as reference, its basically all blocks names with space separator.